lib/fx/function.rb

```diff
@@ -18,7 +18,7 @@ def ==(other)
     def to_schema
       <<-SCHEMA.indent(2)
 create_function :#{name}, sql_definition: <<-\SQL
-#{definition.indent(4).rstrip}
+#{definition.indent(4).rstrip.gsub("\\", "\\\\\\\\")}
```

Comment:
Something about this tells me that this will be error prone.

Is there a generic escape sequence that Rails or any other database related library uses we can reuse here instead?

Reply:
Good push @teoljungberg. The best example I found is from https://github.com/rails/rails/blob/7a321d8429e3d7b9d8aa97a44ce4a4b6cb62de70/actionview/lib/action_view/helpers/javascript_helper.rb#L34

It takes advantage of `gsub`'s ability to accept a hash as the second argument, wherein each match is replaced by the value found by looking up the match in the hash (TIL, https://ruby-doc.org/core-2.7.1/String.html#method-i-gsub) 

What do you think about this implementation?